controller-server/src/main/java/com/yahoo/vespa/hosted/controller/versions/VersionStatus.java

```diff
@@ -130,12 +131,15 @@ public static VersionStatus compute(Controller controller) {
         ListMap<Version, HostName> versions = new ListMap<>();
         for (ZoneId zone : zones) {
             for (SystemApplication application : SystemApplication.all()) {
-                if (!application.configConvergedIn(zone, controller)) {
-                    throw new IllegalStateException("Config for " + application.id() + " in " + zone + " has not converged");
+                boolean configConverged = application.configConvergedIn(zone, controller);
+                if (!configConverged) {
+                    log.log(LogLevel.WARNING, "Config for " + application.id() + " in " + zone + " has not converged");
```

Comment:
Are you sure this is really a WARNING?

Reply:
I suppose it could be made INFO, or what are you suggesting? The point is to log something when this occurs as it may cause the system version to not change.

Reply:
I'd prefer `INFO`, and removal altogether when the infrastructure convergence checker prints these details.

Reply:
Or, ... perhaps leave it `WARNING` for now, and `INFO` at that point.